Specs/DateTime/English/DateTimeModel.json

```diff
@@ -14719,5 +14719,57 @@
         }
       }
     ]
+  },
+  {
+    "Input": "q1 and q2 could be any year",
```

Comment:
Can you just double check that we have cases in these four formats with the year specified and they currently pass? If some is missing, please add a spec, ok?
2019 q1
q3 2019
2019-Q1
q1-2019
Please also add a spec for "2019 q3 and 2020 q1"